Lib/test/test_poplib.py

```diff
@@ -505,17 +505,17 @@ def server(self, evt, serv):
 
     def testTimeoutDefault(self):
         self.assertIsNone(socket.getdefaulttimeout())
-        socket.setdefaulttimeout(30)
+        socket.setdefaulttimeout(test_support.SHORT_TIMEOUT)
```

Comment:
When I added timeout constants to the test.support module, I decided to leave this test unchanged. I understood that the test doesn't connect to real socket. Or does it? I'm not sure. If it does, LOOPBACK_TIMEOUT is preferred.

Reply:
https://github.com/python/cpython/blob/eef1b027ab70704bcaa60a089e4ae1592c504b86/Lib/poplib.py#L107
As you can see poplib client's constructor actually makes a connection to the server and get a response.

https://github.com/python/cpython/blob/eef1b027ab70704bcaa60a089e4ae1592c504b86/Lib/test/test_poplib.py#L484

So this test intends to check whether poplib client is set by a global socket timeout